docs/eventing/channels/create-default-channel.md

```diff
@@ -6,42 +6,61 @@ type: "docs"
 showlandingtoc: "true"
 ---
 
-# Creating a channel using cluster or namespace defaults
+Developers can create channels of any supported implementation type by creating an instance of a
+`Channel` object.
 
-Developers can create channels of any supported implementation type by creating an instance of a Channel object.
+To create a channel:
 
-1. Create a Channel object:
+1. Create a [Channel object](https://knative.dev/docs/reference/api/eventing/#messaging.knative.dev/v1.Channel): <!-- Where do you enter the below? -->
 
       ```yaml
       apiVersion: messaging.knative.dev/v1
       kind: Channel
       metadata:
-        name: my-channel
-        namespace: default
+        name: <my-channel>
+        namespace: <namespace>
       ```
+      Where:
+      * `<my-channel>` is a name of your choice.
```

Comment:
same as above

Reply:
Also I'd say something like "is the name of the channel you want to create" maybe instead

Reply:
Okay, will do